src/Kovan-DssSpell.sol

```diff
@@ -15,71 +15,160 @@
+        DssExecLib.authorize(MCD_END, MCD_ESM);
+
+        // Deuthorize old ESM to execute in old END
+        DssExecLib.deauthorize(MCD_END_OLD, MCD_ESM_OLD);
+
+        // ------------------  AUCTION  ------------------
+
+        ClipperMomLike(CLIPPER_MOM).setAuthority(DssExecLib.getChangelogAddress("MCD_ADM"));
+
+        // Set VOW in the DOG
+        DssExecLib.setContract(MCD_DOG, "vow", MCD_VOW);
+
+        // Set CLIP for LINK-A in the DOG
+        DssExecLib.setContract(MCD_DOG, "LINK-A", "clip", MCD_CLIP_LINK_A);
+
+        // Set VOW in the LINK-A CLIP
+        DssExecLib.setContract(MCD_CLIP_LINK_A, "vow", MCD_VOW);
+
+        // Set CALC in the LINK-A CLIP
+        DssExecLib.setContract(MCD_CLIP_LINK_A, "calc", MCD_CLIP_CALC_LINK_A);
+
+        // Authorize CLIP can access to VAT
+        DssExecLib.authorize(MCD_VAT, MCD_CLIP_LINK_A);
+
+        // Authorize CLIP can access to DOG
+        DssExecLib.authorize(MCD_DOG, MCD_CLIP_LINK_A);
+
+        // Authorize DOG can kick auctions on CLIP
+        DssExecLib.authorize(MCD_CLIP_LINK_A, MCD_DOG);
+
+        // Authorize CLIPPERMOM can set the stopped flag in CLIP
+        DssExecLib.authorize(MCD_CLIP_LINK_A, CLIPPER_MOM);
+
+        // We can't deauthorize the FLIP yet as there might be running auctions:
+        // DssExecLib.deauthorize(MCD_CAT, MCD_FLIP_LINK_A); TODO in a future spell
+
+        // No more auctions kicked via the CAT:
+        DssExecLib.deauthorize(MCD_FLIP_LINK_A, MCD_CAT);
+
+        // No more circuit breaker for the FLIP in LINK-A:
+        DssExecLib.deauthorize(MCD_FLIP_LINK_A, DssExecLib.flipperMom());
+
+        // Authorize the new END to access the LINK CLIP
+        DssExecLib.authorize(MCD_CLIP_LINK_A, MCD_END);
+
+        // Deauthorize the old END from all the FLIPS
+        // Authorize the new END in all the FLIPS
+        // (including MCD_FLIP_LINK_A as there might be still
+        // running auctions and shutdown could happen)
+        bytes32[] memory ilks = IlkRegistryAbstract(ILK_REGISTRY_OLD).list();
+        for (uint256 i = 0; i < ilks.length; i++) {
+            address flip = IlkRegistryAbstract(ILK_REGISTRY_OLD).flip(ilks[i]);
+            DssExecLib.deauthorize(flip, MCD_END_OLD);
+            DssExecLib.authorize(flip, MCD_END);
+        }
+        // DssExecLib.deauthorize(MCD_FLIP_LINK_A, MCD_END); TODO in a future spell
+
+        // Set DOG values
+        Fileable(MCD_DOG).file("Hole", 10 * THOUSAND * RAD);
+        Fileable(MCD_DOG).file("LINK-A", "hole", 10 * THOUSAND * RAD);
+        Fileable(MCD_DOG).file("LINK-A", "chop", 113 * WAD / 100);
+
+        // Set LINK-A CLIP values
+        Fileable(MCD_CLIP_LINK_A).file("buf", 120 * RAY / 100);
+        Fileable(MCD_CLIP_LINK_A).file("tail", 1 hours);
+        Fileable(MCD_CLIP_LINK_A).file("cusp", 60 * RAY / 100);
+        Fileable(MCD_CLIP_LINK_A).file("chip", 2 * WAD / 100);
+        Fileable(MCD_CLIP_LINK_A).file("tip", 50 * RAD);
```

Comment:
Should probably add `clip.upchost()`.

Reply:
Yeah wasn't sure we wanted to do this in the spell, but let's have it for now.